src/deployment-guides/tko-on-azure.md

```diff
@@ -250,18 +250,34 @@ sudo usermod -aG docker $USER
 # docker run hello-world
 
 # Downloading and Installing Tanzu CLI
-git clone https://github.com/z4ce/vmw-cli
-curl -o tmc 'https://tmc-cli.s3-us-west-2.amazonaws.com/tmc/0.4.0-fdabbe74/linux/x64/tmc'
-./vmw-cli/vmw-cli ls vmware_tanzu_kubernetes_grid
-./vmw-cli/vmw-cli cp tanzu-cli-bundle-linux-amd64.tar
-./vmw-cli/vmw-cli cp kubectl-linux-v1.21.2+vmware.1.gz
-
-tar -xvf tanzu-cli-bundle-linux-amd64.tar
-gzip -d kubectl-linux-v1.21.2+vmware.1.gz
 
-sudo install cli/core/v1.4.0/tanzu-core-linux_amd64 /usr/local/bin/tanzu
-tanzu plugin install --local cli all
-sudo install kubectl-linux-v1.21.2+vmware.1 /usr/local/bin/kubectl
+git clone https://github.com/z4ce/vmw-cli
+cd vmw-cli
+curl -o tmc 'https://tmc-cli.s3-us-west-2.amazonaws.com/tmc/0.4.3-fcb03104/linux/x64/tmc'
+./vmw-cli ls
```

Comment:
Is this extra?

Reply:
I believe it is required the first time you use the vmw-cli to populate the cache